admin-app/src/shared/chat-list/components/chat-list-widget/styled.ts

```diff
@@ -1,9 +1,51 @@
 import styled from "styled-components";
 import { color } from "../../../../constants/colors";
 
+export const ChatListWrapper = styled.div`
+  max-width: 280px;
```

Comment:
i think better use flex property, like flex; 1 1 280px; or something else, like wrapper with max-width and flex 1 1 auto